### DISCUSSION TITLE
About 15% to 20% of the plot files started from version 1.2.6 to version 1.2.9, and the harvester could not be found and could not be farmed. But when you run the chia plots check -l command, these files still exist.

### DISCUSSION THREAD

**Discussion by alex826-pex** [Chia-Network/chia-blockchain#8794] [Support]
About 15% to 20% of the plot files started from version 1.2.6 to version 1.2.9, and the harvester could not be found and could not be farmed. But when you run the chia plots check -l command, these files still exist.
My operating system is ubunutu 20
A total of 6 storage servers have this problem, and other servers are operating normally.

**Comment by BrandtH22** [comment]
@alex826-pex , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
